js/woo.js

```diff
@@ -679,13 +680,22 @@ module.exports = class woo extends Exchange {
          * @param {object} params extra parameters specific to the woo api endpoint
          * @returns {object} an [order structure]{@link https://docs.ccxt.com/en/latest/manual.html#order-structure}
          */
+        const reduceOnly = this.safeValue (params, 'reduceOnly');
+        if (reduceOnly !== undefined) {
+            if (type.toUpperCase () !== 'LIMIT') {
```

Comment:
@sc0Vu we don't have to do `type.toUpperCase()` twice in this function, let's deduplicate it.